sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -47,8 +47,10 @@
   SchemaField = None
   NotFound = None
 
-RUNTIME_METRIC = 'runtime'
-COUNTER_LABEL = 'total_bytes_count'
+METRICS = {
```

Comment:
I'm a bit concerned about this nested dictionary. It's fine if there will be more metrics types (aside from counters and distributions) in the future. But if not, it would be easier if you split this into two dictionaries.

Reply:
Well, there is possibility we will have gauges as well.